src/ScriptCs.Core/ReplCommands/ScriptPacksCommand.cs

```diff
@@ -0,0 +1,185 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using ScriptCs.Contracts;
+using ScriptCs.Extensions;
+
+namespace ScriptCs.ReplCommands
+{
+    public class ScriptPacksCommand : IReplCommand
+    {
+        private readonly IConsole _console;
+
+        public ScriptPacksCommand(IConsole console)
+        {
+            _console = console;
+        }
+
+        public string Description
+        {
+            get { return "Displays information about script packs available in the REPL session"; }
+        }
+
+        public string CommandName
+        {
+            get { return "scriptpacks"; }
+        }
+
+        public object Execute(IRepl repl, object[] args)
+        {
+            var packContexts = repl.ScriptPackSession.Contexts;
+            var originalColor = _console.ForegroundColor;
+            _console.ForegroundColor = ConsoleColor.Yellow;
+
+            if (packContexts.IsNullOrEmpty())
+            {
+                _console.WriteLine("There are no script packs available in this REPL session");
+                _console.ForegroundColor = originalColor;
+                return null;
+            }
+
+            var importedNamespaces = repl.ScriptPackSession.Namespaces.IsNullOrEmpty() ? repl.Namespaces.ToArray() : repl.Namespaces.Union(repl.ScriptPackSession.Namespaces).ToArray();
+
+            foreach (var packContext in packContexts)
+            {
+                var contextType = packContext.GetType();
+                
+                _console.WriteLine(contextType.ToString());
+                _console.ForegroundColor = originalColor;
```

Comment:
Because the yellow colour is set outside this loop, the first script pack name will be printed in yellow, but every subsequent one will be printed in green. The yellow should be set just before writing out the name of the script pack.

Also, since this a static state mutation, after setting the yellow, the `WriteLine` should really be a in a `try{}` with the resetting of the colour in a finally.

(Once I pull my finger out and produce a source package for https://github.com/colored-console/colored-console, we can use it to wrap up colour management for us.)